models/article.model.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection");
+
+exports.selectArticleById = (article_id) => {
+  let baseSQLString = `SELECT * FROM articles `;
+  if (article_id) {
```

Comment:
You can have the query without the if statement like: let baseSQLString = `SELECT * FROM articles WHERE article_id = $1` We want to build the queries like this only when there are variable situations, in this case you always want to query with an id 🙂